R/draw_legislaturelabels.R

```diff
@@ -0,0 +1,182 @@
+#' Draw total number of seats in the middle of the parliament
+#' @param n  The number of total seats
+#' @param type Type of parliament - supports "horseshoe" or "semicircle"
+#' @examples
+#' data <- ggparliament::election_data %>% filter(year == "2016" & country == "USA" & house == "Representatives")
+#' usa_data <- parliament_data(election_data = data, type = "semicircle", party_seats = data$seats, party_names = data$party_short, parl_rows = 8, total_seats = sum(data$seats))
+#' ggplot(usa_data, aes(x, y, color=party_long)) + geom_parliament_seats() + draw_totalseats(n = 435, type = 'semicircle') + theme_void()
+#' @author Zoe Meers
+#' @export
+
+
+draw_totalseats <- function(n = NULL,
```

Comment:
this is obviously just an aesthetic change, but I wonder if we should standardise function names. I would propose using geom_ for anything that's a shape/coloured, label_ for these funcs and so on? Will leave unchanged for now